classes/task/insert_recordings.php

```diff
@@ -54,7 +54,7 @@ public function execute() {
             } catch (\moodle_exception $error) {
                 mtrace('No recordings found for the meeting_id: '. $value->meeting_id);
             }
-             if (!empty($recordings) && !empty($recordings->recording_files{0})) {
+            if (!empty($recordings) && !empty($recordings->recording_files{0})) {
```

Comment:
Can we have this entire block inside try-catch block ???

Reply:
@Naveen1895 code changed and updated.